addons/web/static/lib/pdfjs/web/viewer.js

```diff
@@ -18255,7 +18255,7 @@ window.print = function () {
       return activeServiceOnEntry.performPrint();
     }).catch(function () {}).then(function () {
       if (activeServiceOnEntry.active) {
-        abort();
+        setTimeout(abort, 1000); // ODOO Patch
```

Comment:
I added the reference of why we added this patch.
Since our team is responsible for updating the libraries, we systematically search for "Odoo" word before processing.